recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-bsp/arm-trusted-firmware/files/0001-rcar-Use-UART-instead-of-Secure-DRAM-area-for-loggin.patch

```diff
@@ -36,15 +36,15 @@ index 4178848..a711998 100644
 +endfunc console_uninit
 +
 diff --git a/plat/renesas/rcar/platform.mk b/plat/renesas/rcar/platform.mk
-index 78db0eb..3086f09 100644
+index a8b4672..27480fc 100644
 --- a/plat/renesas/rcar/platform.mk
 +++ b/plat/renesas/rcar/platform.mk
-@@ -69,7 +69,7 @@ BL31_SOURCES		+=	${RCAR_GIC_SOURCES}				\
- 				plat/renesas/rcar/rcar_pm.c			\
+@@ -70,7 +70,7 @@ BL31_SOURCES		+=	${RCAR_GIC_SOURCES}				\
  				plat/renesas/rcar/rcar_sip_svc.c		\
+ 				plat/renesas/rcar/drivers/board/board.c		\
  				plat/renesas/rcar/drivers/dramconf/dramconf.c	\
 -				plat/renesas/rcar/drivers/memdrv/rcar_console.S	\
-+				plat/renesas/rcar/drivers/scif/scif.S		\
++				plat/renesas/rcar/drivers/scif/scif.S			\
```

Comment:
I believe, this is spurious change. No need for tab symbol there.

Reply:
updated